PolyEngine/Engine/Src/Physics3DWorldComponent.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <map>
+
+#include "ComponentBase.hpp"
+#include "Physics3DSystem.hpp"
+
+class btDiscreteDynamicsWorld;
+class btDefaultCollisionConfiguration;
+class btCollisionDispatcher;
+class btBroadphaseInterface;
+class btSequentialImpulseConstraintSolver;
+class btCollisionObject;
+
+namespace Poly
+{
+	/// Config for physics simulation
+	/// @see Physics3DWorldComponent
+	/// @see Trigger3DComponent
+	/// @see Rigidbody3DComponent
+	struct ENGINE_DLLEXPORT Physics3DConfig
+	{
+	public:
+		float TimeStep = 1.f / 60.f;
+		Vector Gravity = Vector(0.f, -9.81f, 0.f);
+		float TimeFactor = 1.f;
```

Comment:
What is time factor? We don't really need to speedup/slowdown simulation. We can do it via timers manipulation.